test/fixtures/remote-error-test-cases/get-jstp-array.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
@aqrln Thank you for your review.
I have added changes due to it, although I do not see a lot of imperative logic:
* scanning for default error messages
* calling a `RemoteError` and `TypeError` constructor

Both moved to a related test.